util/src/main/java/tech/pegasys/artemis/util/config/NetworkDefinition.java

```diff
@@ -42,8 +42,20 @@
                       "https://github.com/eth2-clients/eth2-testnets/raw/master/prysm/Topaz(v0.11.1)/genesis.ssz")
                   .discoveryBootnodes(
                       "enr:-Ku4QAGwOT9StqmwI5LHaIymIO4ooFKfNkEjWa0f1P8OsElgBh2Ijb-GrD_-b9W4kcPFcwmHQEy5RncqXNqdpVo1heoBh2F0dG5ldHOIAAAAAAAAAACEZXRoMpAAAAAAAAAAAP__________gmlkgnY0gmlwhBLf22SJc2VjcDI1NmsxoQJxCnE6v_x2ekgY_uoE1rtwzvGy40mq9eD66XfHPBWgIIN1ZHCCD6A")
-                  .eth1DepositContractAddress(
-                      Eth1Address.fromHexString("0x5cA1e00004366Ac85f492887AAab12d0e6418876"))
+                  .eth1DepositContractAddress("0x5cA1e00004366Ac85f492887AAab12d0e6418876")
+                  .build())
+          .put(
+              "schlesi",
+              builder()
+                  .constants("mainnet")
```

Comment:
Does this honor the custom genesis time, min genesis delay, and min validator count?

Reply:
Since we're running from a supplied genesis state, those constants shouldn't have any effect (genesis time, min genesis delay, and min validator count).

Reply:
We probably should switch it to using https://github.com/goerli/schlesi/raw/master/teku/chain.yaml anyway.  

It worries me a bit that the genesis and constants definitions for Schlesi are different files for each client - they should be exactly the same if we want the network to work.

Reply:
Right, they differ a bit because not all clients honor all constants yet.